lib/runtime/evaluation.coffee

```diff
@@ -16,19 +16,21 @@ module.exports =
     edpath = editor.getPath() || 'untitled-' + editor.getBuffer().id
     fn {editor, mod, edpath}
 
-  eval: ({move}={}) ->
+  eval: ({move, cell}={}) ->
```

Comment:
Or not. Would possibly be cleaner to have a common API for `blocks` and `cells` and then either call the methods from one or the other, i.e. 
```
selector = if cell? then cells else blocks
selector.get(editor)
...
selector.moveNext editor, ...
```